manager/manager.go

```diff
@@ -671,7 +671,7 @@ func (m *manager) GetProcessList(containerName string, options v2.RequestOptions
 	// TODO(rjnagal): handle count? Only if we can do count by type (eg. top 5 cpu users)
 	ps := []v2.ProcessInfo{}
 	for _, cont := range conts {
-		ps, err = cont.GetProcessList()
+		ps, err = cont.GetProcessList(m.cadvisorContainer == "/")
```

Comment:
Sadly I don't think this will work since cAdvisor may not be running inside namespaces (like in Kubernetes). We should check the mount/pid namespace of us vs init?